tensorflow_addons/seq2seq/beam_search_ops_test.py

```diff
@@ -89,12 +86,14 @@ def testBadParentValuesOnGPU(self):
         expected_result = _transpose_batch_time([[[2, -1, 2], [6, 5, 6],
                                                   [7, 8, 9], [10, 10, 10]]])
         with tf.device("/device:GPU:0"):
-            beams = gather_tree(
-                step_ids=step_ids,
-                parent_ids=parent_ids,
-                max_sequence_lengths=max_sequence_lengths,
-                end_token=end_token)
-            self.assertAllEqual(expected_result, self.evaluate(beams))
+            msg = r"parent id -1 at \(batch, time, beam\) == \(0, 0, 1\)"
+            with self.assertRaisesOpError(msg):
+                beams = gather_tree(
+                    step_ids=step_ids,
+                    parent_ids=parent_ids,
+                    max_sequence_lengths=max_sequence_lengths,
+                    end_token=end_token)
+                self.assertAllEqual(expected_result, self.evaluate(beams))
```

Comment:
I don't think this change is correct. The error message is only thrown in the CPU implementation, but not GPU implementation. The raise of this error message means the cudnn kernel is not correctly invoked.

Reply:
ohoh, I got it. Will invoke it later. But wonder if the implementation on GPU and CPU should raise the same error message.